### ISSUE TITLE
Use Task level attributes instead of folder

### ISSUE THREAD

**Issue by MustafaJafar** [ynput/ayon-houdini#14]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior:

Houdini addon uses folder level attributes e.g. frame range

### Expected Behavior:

It should use task level attributes.

### Version

1.0.0

### What platform you are running on?

Windows

### Steps To Reproduce:

This is an example,
1. On AYON Server, change frame range on task
2. In Houdini, use `Set Frame Range`, frame range will be reset to folder frame range not task.

### Are there any labels you wish to add?

- [X] I have added the relevant labels to the bug report.

**Comment by MustafaJafar** [comment]
Questions: 
1. Could some one provide me a list of attributes ?
- fps
- resolution
- frame start
- frame end

2. Should we have toggle to switch between task an folder attributes ?

**Comment by BigRoy** [comment]
> Should we have toggle to switch between task an folder attributes ?

No, in a recent similar PR for Max or Maya I believe it came up that these attributes must always exist on tasks (and inherit by default from parents). So we should be able to rely on these from just the task entity. @iLLiCiTiT confirmed that somewhere, but I can't find it.

I think we're looking for these attributes to be available like that on Task entities:

![image](https://github.com/ynput/ayon-houdini/assets/2439881/caa54082-47d8-4881-bf69-00b3e896827f)

**Comment by iLLiCiTiT** [comment]
Confirming again. We should use only task attributes (if task is set). Task inherit attribute values from parent folder.
